web/templates/confirm_dialog/confirm_deactivate_user.hbs

```diff
@@ -42,3 +42,41 @@
         <textarea class="email_field_textarea settings_textarea" rows="8" maxlength="2000"></textarea>
     </div>
 </div>
+<br/>
+<label class="checkbox">
+    <input id="is-user-spammer" type="checkbox" class="spammer" />
+    <span class="rendered-checkbox"></span>
+    {{t "Is this user a spammer?" }}
+</label>
+<div class="spammer-actions">
+    <p class="border-top">{{t 'Upon deactivation:'}}</p>
+    <ul class="spammer-actions-body">
+        <li>
+            {{#tr}}
+            They will be renamed to <strong>Deleted user</strong>.
+            {{/tr}}
+        </li>
+        <li>{{t "Their profile picture will be removed."}}</li>
+    </ul>
+    {{#if user_can_delete_any_message}}
+    <div class="delete-message-handling">
+        <label for="message-action">
```

Comment:
I guess `for` attribute can be removed here since it is set to the ID of the element it is referring but here we do not have ID for the input elements and there are multiple input elements.